### PR TITLE
NAS-3911 updated clients to the latest version

### PR DESCRIPTION
**Pull request by hkad98**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: docker-compose.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ version: '3.7'
 services:
   gooddata-cn-ce:
     # Use stable releases represented by tags like 1.5
-    image: gooddata/gooddata-cn-ce:1.7.2
+    image: gooddata/gooddata-cn-ce:dev_latest
```
Comment (janchochol) [inline docker-compose.yaml:10]: This does not look consistent with comment one line above.

Comment (hkad98) [reply docker-compose.yaml:10]: True, thanks. I will fix it 👍

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by jaceksan — APPROVED** [review]

**Review by pcerny — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR109

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/101/ : SUCCESS in 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul-el7/101/ : SUCCESS in 2m 59s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/101/ : SUCCESS in 26s

**Pull request merged**